internal/endpoints/config/go.mod

```diff
@@ -32,6 +32,8 @@ replace github.com/aws/aws-sdk-go-v2/internal/endpoints/v2 => ../../../internal/
 
 replace github.com/aws/aws-sdk-go-v2/internal/ini => ../../../internal/ini/
 
+replace github.com/aws/aws-sdk-go-v2/service/internal/accept-encoding => ../../../service/internal/accept-encoding/
```

Comment:
clarifying question: is this `accept-encoding` change part of identity and auth?

Reply:
There's no change to speak of, this must have been some intermediate artifact of codegen during development. The module in question isn't used here nor in the other places that the replace statement has appeared. I'll remove it.

Reply:
the mod replace command is recursive. so an indirect dependency might be using it